org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/VersionUtilities.java

```diff
@@ -778,4 +773,24 @@ public static String getResourceTypesUrl(String version) {
   }
 
 
+  public static List<String> iterateCoreVersions(String startVer, String endVer) {

```

Comment:
getCoreVersionRange seems like a better description here. And a minor nitpick; this could be an immutable list.

Reply:
I don't think so - it gets given a range, and iterates the list inside the range